libnavigation-base/src/main/java/com/mapbox/navigation/base/options/EHorizonOptions.kt

```diff
@@ -40,10 +42,10 @@ class EHorizonOptions private constructor(
 
         other as EHorizonOptions
 
-        if (length != other.length) return false
+        if (length.notEquals(other.length)) return false
```

Comment:
https://jqno.nl/equalsverifier/errormessages/double-equals-doesnt-use-doublecompare-for-field-foo/